harmony-fe/src/pages/_app.tsx

```diff
@@ -17,6 +17,7 @@ type AppPropsWithLayout = AppProps & {
 
 export default function App({ Component, pageProps }: AppPropsWithLayout) {
   const [currentUser, setCurrentUser] = useState<User | undefined>(undefined)
+  const [userCookie, setUserCookie] = useState<string>('')
```

Comment:
In the next sprint, lets pop some of this logic out into other files and import it here

Reply:
Yeah, I can see it getting cluttered real quick